startup/development.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.6.6-alpine3.12
+
+RUN apk add --no-cache --update git
+WORKDIR /usr/src/
+RUN git clone https://github.com/exercism/config.git exercism_config
+WORKDIR /usr/src/exercism_config
+
+# COPY Gemfile Gemfile.lock ./
+RUN gem install bundler:2.1.4 && \
+    bundle install
+
+ENV EXERCISM_CONFIG_VERSION 0.26.0
```

Comment:
This still needs removing
```suggestion
```

Reply:
Per discussion last night we'll get this as an argument from docker-compose.yml and you were going to tag releases on Github (which you should probably do in any case IMHO).  When you add 0.29.0 I can test this again and push a final update.